wiki/Guides/OpenGL_Support_Issues/en.md

```diff
@@ -13,7 +13,7 @@ First, let's check if this applies to you:
 
 Check whether you are using Microsoft Basic Display Adapter as per the following diagram:
 
-![Device Manager](img/Devicemanager.png "Device Manager")
+![Device Manager](img/devicemanager.png "Device Manager")
```

Comment:
you forgot to rename this file

Reply:
Sorry, but I'm not sure I understand what you mean. Are you refering to the file format (jpg instead of png)? Should it be `device-manager.png`?

Reply:
you changed this text here from `Devicemanager.png` to `devicemanager.png` but forgot to actually rename the `img/Devicemanager.png ` file

![image](https://user-images.githubusercontent.com/36758269/146410106-d86a4b92-9cef-4822-b1ad-ed816e110d17.png)

Reply:
![image](https://user-images.githubusercontent.com/50201659/146411911-7dbc3e35-adc4-4999-b8e1-052612161063.JPG)

It's showing up as `devicemanager.png` in my local file system. Git also says that there's nothing to push. Weird 🤔

Reply:
i've fixed it for you. had to rename the file to something else so git would recognise it and then rename it again O.o